Add documentation links

BenConstable9 opened the pull request:
## Summary of the Pull Request

**What is this about:**

Add documentation links to the system tray and the settings menu. I was looking for an easy PR to work on so hopefully this is okay.

![image](https://user-images.githubusercontent.com/15877083/128498484-f49c439c-ea6b-4c08-b320-03e1b9bf26c3.png)
![image](https://user-images.githubusercontent.com/15877083/129278514-aa825726-0caa-4b0b-ba09-b8e1d12720e1.png)

**What is include in the PR:** 

Overview link placed in the application

**How does someone test / validate:** 

Manual validation by pressing the system tray menu item

## Quality Checklist

- [x] **Linked issue:** #9515
- [ ] **Communication:** I've discussed this with core contributors in the issue. 
- [ ] **Tests:** Added/updated and all pass
- [ ] **Installer:** Added/updated and all pass
- [x] **Localization:** All end user facing strings can be localized
- [ ] **Docs:** Added/ updated
- [ ] **Binaries:** Any new files are added to WXS / YML
   - [ ] No new binaries
   - [ ] [YML for signing](https://github.com/microsoft/PowerToys/blob/master/.pipelines/pipeline.user.windows.yml#L68) for new binaries
   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/master/installer/PowerToysSetup/Product.wxs) for new binaries

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.

BenConstable9 committed:
Add documentation links

htcfreek reviewed (commented):
Thank you for your contribution.

I suggest a small grammar related text change.

BenConstable9 committed:
Update src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

Co-authored-by: Heiko <61519853+htcfreek@users.noreply.github.com>

BenConstable9 committed:
Update src/runner/Resources.resx

Co-authored-by: Heiko <61519853+htcfreek@users.noreply.github.com>

BenConstable9 commented:
> Thank you for your contribution.
> 
> I suggest a small grammar related text change.

I have made those changes.

htcfreek commented:
@BenConstable9 
Can you please add two screenshots (tray and settings) to the PR description. Thank you.

jaimecbernardo commented:
Hi @htcfreek,
Here are some screenshots:
![image](https://user-images.githubusercontent.com/26118718/128247125-b8407d56-cdf4-4ec7-8d6e-fc3ad651faaa.png)

![image](https://user-images.githubusercontent.com/26118718/128247202-e1504216-82c6-483a-873d-1ae608c9df67.png)

htcfreek commented:
@jaimecbernardo 
Let's switch docs and GitHub in settings window.

jaimecbernardo commented on src/runner/tray_icon.cpp:
```diff
@@ -83,6 +83,9 @@ void handle_tray_command(HWND window, const WPARAM command_id)
         }
         break;
+    case ID_DOCUMENTATION_MENU_COMMAND:
+        ShellExecuteA(NULL, "open", "https://aka.ms/PowerToysOverview", NULL, NULL, SW_SHOWNORMAL);
```
Should be `SellExecuteW` instead of `ShellExecuteA`, to be consistent with the rest of the code.

The main issue here is that this may open elevated browser windows from the tray icon when PowerToys is running as an Administrator. Since we are accepting commands from non elevated processes to be able to open settings, I imagine this might be used as an attack vector.

Could you please see if you can include `#include <common/utils/elevation.h>` and try to do `RunNonElevatedEx(L"https://aka.ms/PowerToysOverview", L"");` instead? That should make it so the browser windows start non-elevated.

BenConstable9 replied:
Thanks for this. It is something I was not aware of but very useful to know! I've made the changes.

jaimecbernardo reviewed (changes requested):
Thank your for the contribution.
I've added some comments regarding a possible security issue.
Other than that, LGTM.

jaimecbernardo commented:
> @jaimecbernardo 
> Let's switch docs and GitHub in settings window.

@BenConstable9 , please take this into consideration as well.

BenConstable9 committed:
Swap order of repo and documentation

BenConstable9 committed:
Fix potential security issue

BenConstable9 commented:
> > @jaimecbernardo
> > Let's switch docs and GitHub in settings window.
> 
> @BenConstable9 , please take this into consideration as well.

All fixed now. Thanks!

htcfreek commented:
@BenConstable9 
Can you please add updated screenshots to the original PR description.

BenConstable9 commented:
@htcfreek updated the description

htcfreek commented:
@BenConstable9 
What do you think on adding a line between settings and doc in tray menu.

BenConstable9 committed:
Merge branch 'master' into system-tray-doc-link

BenConstable9 committed:
Add seperator

BenConstable9 committed:
Merge branch 'system-tray-doc-link' of https://github.com/BenConstable9/PowerToys into system-tray-doc-link

BenConstable9 committed:
Fix failing test and build

BenConstable9 commented:
> @BenConstable9
> What do you think on adding a line between settings and doc in tray menu.

@htcfreek I think it looks better. I've added the line and fix the fact the build was failing. Updated screenshot in PR description

htcfreek reviewed (approved):
@BenConstable9 
Screenshots looking good to me. Thanks for the contribution.
Code review has to be done by other main contributors. (cc: @enricogior, @yuyoyuppe)

jaimecbernardo reviewed (approved):
LGTM! Thank you for the contribution!
